x/withdraw/types/params.go

```diff
@@ -0,0 +1,76 @@
+package types
+
+import (
+	"fmt"
+	"time"
+
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter store key
+var (
+	ParamStoreKeyEnableWithdraw        = []byte("EnableWithdraw")
+	ParamStoreKeyPacketTimeoutDuration = []byte("PacketTimeoutDuration")
+)
+
+// ParamKeyTable returns the parameter key table.
+func ParamKeyTable() paramtypes.KeyTable {
+	return paramtypes.NewKeyTable().RegisterParamSet(&Params{})
+}
+
+// NewParams creates a new Params instance
+func NewParams(
+	enableWithdraw bool, timeoutDuration time.Duration,
+) Params {
+	return Params{
+		EnableWithdraw:        enableWithdraw,
+		PacketTimeoutDuration: timeoutDuration,
+	}
+}
+
+// DefaultParams defines the default params for the withdraw module
+func DefaultParams() Params {
+	return Params{
+		EnableWithdraw:        true,
+		PacketTimeoutDuration: 4 * time.Hour,
```

Comment:
why 4 hours? standard?

Reply:
@colin-axner suggested that 1 hr was prob too low

Reply:
could leave a comment about it